examples/models/resnet.py

```diff
@@ -0,0 +1,51 @@
+import torch.nn as nn
+from models.modules.layers_3d import SparseConvBlock, SparseResBlock
+
+__all__ = ['SparseResNet']
+
+
+class SparseResNet(nn.Module):
```

Comment:
The current model is `SparseResNet18`. We could think about supporting general `SparseResNet` in the future.